plugins/serializers/value/value.go

```diff
@@ -0,0 +1,14 @@
+package value
+
+import (
+	"fmt"
+	"github.com/influxdata/telegraf"
+)
+
+type ValueSerializer struct {
+}
+
+func (s *ValueSerializer) Serialize(metric telegraf.Metric) ([]byte, error) {
+	result := fmt.Sprintf("%v", metric.Fields()["value"])
```

Comment:
I think this will print `<nil>`, if there is no value field I think it should return `[]byte("")`.